GameData/RealismOverhaul/RO_SuggestedMods/CryoEngines/RO_CryoEngines_Engines.cfg

```diff
@@ -217,4 +217,160 @@
 
 		!CONFIG[*]:HAS[~name[RL10A-5]] {}
 	}
+}
+
+//=============
+//	Raptor SL
+//=============
+
+@PART[cryoengine-deinonychus-1]:FOR[RealismOverhaul]
+{
+	%RSSROConfig = True
+	@rescaleFactor = 1.557
+	
+	%maxTemp = 1700
+	%crashTolerance = 12
+	%breakingForce = 250
+	%breakingTorque = 250
```

Comment:
Please break these out into a common patch:
```
@PART[a,b,c,d,e]:FOR[RealismOverhaul]
{
	%maxTemp = 1700
	%crashTolerance = 12
	%breakingForce = 250
	%breakingTorque = 250
}
```
Also suggest using the defaults configured in ROE [and which perhaps we should make the default settings within RO?]  <https://github.com/KSP-RO/ROEngines/blob/master/GameData/ROEngines/ROEEngineDefaults.cfg>
```
	%crashTolerance = 10
	%maxTemp = 673.15
	%skinMaxTemp = 773.15
	%fuelCrossFeed = true
	%breakingForce = 10000
	%breakingTorque = 10000
```

Reply:
Same with this.